web/app/models/DrivingSession.js

```diff
@@ -0,0 +1,14 @@
+// grab the mongoose module
+var mongoose = require("mongoose");
+
+// define our driving session model
+// module.exports allows us to pass this to other files when it is called
+var DrivingSessionSchema = new mongoose.Schema({
+    startTime: { type: String, required: true },
```

Comment:
Why strings? Mongoose supports a Date type which is basically a time stamp.  I feel strings will be very difficult to deal with.